src/confetti.js

```diff
@@ -334,6 +358,8 @@
       context.ellipse ?
         context.ellipse(fetti.x, fetti.y, Math.abs(x2 - x1) * fetti.ovalScalar, Math.abs(y2 - y1) * fetti.ovalScalar, Math.PI / 10 * fetti.wobble, 0, 2 * Math.PI) :
         ellipse(context, fetti.x, fetti.y, Math.abs(x2 - x1) * fetti.ovalScalar, Math.abs(y2 - y1) * fetti.ovalScalar, Math.PI / 10 * fetti.wobble, 0, 2 * Math.PI);
+    } else if (fetti.shape === 'star') {
+      drawStar(context, fetti.x, fetti.y, 5, 5, 10);
```

Comment:
I am not sure where exactly, but `fetti.scalar` should be used somewhere to allow the stars to grow along with the other shapes (you can see the `scalar` value in the `options` documentation. My guess would be that it can be included here... perhaps multiplying the inner and outer radius by the scalar value?